models/h2o_dai_german_credit/app_h2o_mojo_pipeline.py

```diff
@@ -0,0 +1,48 @@
+from certifai.model.sdk import SimpleModelWrapper
+import pandas as pd
+import numpy as np
+from numpy import nan
+import datatable as dt
+import daimojo.model
+
+model = daimojo.model("./pipeline.mojo")
+print(f"Loaded {model.uuid} from ./pipeline.mojo")
+
+def _get_prediction_class(preds):
+    if preds[0] > preds[1]:
+        return 1
+    return 2
+
+columns = ['checkingstatus',
+    'duration',
+    'history',
+    'purpose',
+    'amount',
+    'savings',
+    'employ',
+    'installment',
+    'status',
+    'others',
+    'residence',
+    'property',
+    'age',
+    'otherplans',
+    'housing',
+    'cards',
+    'job',
+    'liable',
+    'telephone',
+    'foreign'
+]
+
+class GermanCredit(SimpleModelWrapper):
+    def predict(self, npinstances):
+        instances = [tuple(instance) for instance in npinstances]
+        input_dt = dt.Frame(instances, names=columns)
+        predictions = model.predict(input_dt).to_pandas().apply(_get_prediction_class, axis=1)
+        return predictions.values
+
+if __name__ == "__main__":
+    # Host is set to 0.0.0.0 to allow this to be run in a docker container
+    app = GermanCredit(host="0.0.0.0")
+    app.run(log_level="warn")
```

Comment:
this is just ignored since its not a gunicorn server. no harm to keep it though

Reply:
The 'log_level'? That makes sense. I was _trying_ to get the noise out of certifai.log, but failed. Created a separate ticket to deal with that as it really makes it hard to check for scan errors.